src/main/webapp/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>capstone project</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyB2RvKUmgsgzAyEten9CPEKwO7SdLQuz40"></script>
+    <script src="script.js"></script>
+  </head>
+  <body onload="">
+    <h1> FoodMood - your food order helper!</h1>
+    <form action="/action_page.php">
+      <label for="cuisine">Choose a cuisine:</label>
+      <select class="button" name="cuisine" id="cuisine">
+        <option value="all">all cuisines</option>
+        <option value="Italian">Italian</option>
+        <option value="Asian">Asian</option>
+        <option value="Sushi">Sushi</option>
+        <option value="Hamburger">Hamburger</option>
+        <option value="Salad">Salad</option>
+        <option value="Homemade">Homemade</option>
+      </select>
+      <br><br>
+      <label for="rating">Min rating:</label>
+      <select class="button" name="rating" id="rating">
+        <option value="5">5 stars out of 5</option>
+        <option value="4" selected="selected">4 stars out of 5</option>
+        <option value="3">3 stars out of 5</option>
+        <option value="2">2 stars out of 5</option>
+        <option value="1">1 stars out of 5</option>
+      </select>
+      <br><br>
+      <label for="price">price range:</label>
+      <select class="button" name="price" id="price">
+        <option value="low">low budget</option>
+        <option value="normal" selected="selected">normal</option>
+        <option value="high">wiiling to spend a little extra</option>
+        <option value="very high">no budget limit</option>
+      </select>
+      <br><br>
+      <input class="submit" type="submit" value="Decide for me!">
```

Comment:
Pressing "submit" currently breaks the page. Maybe you can avoid that for now by disabling the submit action?

Reply:
great comment, thanks!